src/app/checkout/step-2/existingPaymentMethods/existingPaymentMethods.tpl.html

```diff
@@ -3,9 +3,9 @@
   <div class="panel-body">
     <div class="radio radio-method" ng-repeat="paymentMethod in $ctrl.paymentMethods" ng-init="expired = !$ctrl.validPaymentMethod(paymentMethod)">
       <label>
-        <input ng-if="!expired" type="radio" name="paymentMethod" ng-model="$ctrl.selectedPaymentMethod" ng-value="paymentMethod" required>
-        <button ng-if="expired" class="btn btn-xs btn-default" ng-click="$ctrl.openPaymentMethodFormModal(paymentMethod)">Update</button>
+        <input type="radio" name="paymentMethod" ng-model="$ctrl.selectedPaymentMethod" ng-value="paymentMethod" ng-disabled="expired" required>
         <payment-method-display payment-method="paymentMethod" expired="expired"></payment-method-display>
+        <button class="btn btn-xs btn-link" ng-click="$ctrl.openPaymentMethodFormModal(paymentMethod)" translate>update</button>
```

Comment:
Idk that users should have to think about what the difference between edit and update are. I changed them all to edit. While I personally think update sounds better, ("update an expired card", "update a mistyped CVV", "update my card's billing address after I moved") we are using the edit terminology everywhere else.